src/lib/logic-function.js

```diff
@@ -214,6 +214,17 @@ class LogicFunction {
 		}
 	}
 
+	copyFromOtherLogicFunction(logicFunction) {
+		this.name = logicFunction.name;
+		this.org = logicFunction.org;
+		this.description = logicFunction.description;
+		this.enabled = logicFunction.enabled;
+		this.type = logicFunction.type;
+		this.triggers = logicFunction.triggers;
+		this.files.sourceCode.content = logicFunction.files.sourceCode.content;
+		this.files.configuration.content = logicFunction.files.configuration.content;
+	}
+
```

Comment:
what is the `copyFromOtherLogicFunction` for?

Reply:
is used to "sync" a logicFunction from another one. Specifically is used to copy values from "cloud" to "local" but is more generic than that.